test/features/datasets.feature

```diff
@@ -5,3 +5,18 @@ Feature: Dataset APIs
         When I log in
         And I edit the "warandpeace" dataset
         Then I should see an element with xpath "//option[@value='cc-by-nc-sa-4']"
+
+    Scenario: As a user with publishing privileges, I can create a dataset
+        Given "TestOrgEditor" as the persona
```

Comment:
Create another user as you renamed Publisher to Editor to bypass the bug.

Reply:
This isn't meant to test the username validation. It's just a sanity-check that we can create a new dataset, since we're altering the routing in a way that has the potential to break `/dataset/new` (https://dev.data.qld.gov.au/dataset/new is broken right now).